tests/console/zypper_info.pm

```diff
@@ -47,39 +47,36 @@ sub run {
 
     # preparation of source repositories
     my $cmd;
-    if (check_var('DISTRI', 'sle')) {
+    if (is_sle) {
+        if (sle_version_at_least('15') && get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE')) {
```

Comment:
Let's see if we need to add source repo anywhere else and then can extract this logic to some utils.